conda_build/main_build.py

```diff
@@ -30,6 +30,7 @@
     'numpy': [16, 17, 18, 19, 110],
     'perl': None,
     'R': None,
+    'lua': ["2.0", "5.1", "5.2", "5.3"]
```

Comment:
I think the `.` could have implications for how conda parses package names.  Probably best to follow the pattern already established for python and numpy.